### DISCUSSION TITLE
HELP!WSA failed

### DISCUSSION THREAD

**Discussion by TruddJump** [LSPosed/MagiskOnWSALocal#456] [Q&A]
When I run the run.bat,after install WSA failed to start.(ARM64)
while there's 0 app installed in my WSA.
I've uninstalled and reinstalled it but it doesn't work.
When I execute the WSAsetting in the folder it shows an error which said MSVCP140_APP.dll losted.Will it influence the WSA?
HELP!!!

**Comment by Almighty188** [comment]
Which brand of GApps are you using?  OpenGApps or MindTheGapps?
